app/components/Edit/EditScheduleDay.js

```diff
@@ -12,31 +13,52 @@ class EditScheduleDay extends Component {
     return (
       <li key={index}>
         <p>{ index === 0 ? abbrev : '' }</p>
-        <div>
-          <input type="time" defaultValue={this.props.getDayHours(day, "opens_at", index)} data-key={day} data-field="opens_at" onChange={(e) => this.props.handleScheduleChange(day, index, "opens_at", e.target.value)}/>
-          <input type="time" defaultValue={this.props.getDayHours(day, "closes_at", index)} data-key={day} data-field="closes_at" onChange={(e) => this.props.handleScheduleChange(day, index, "closes_at", e.target.value)}/>
-          {index > 0 ? <button onClick={() => this.props.removeTime(day, index)}className="remove-time">x</button> : ''}
-        </div>
+        {is24Hours ?
+          <div>
+            <p className="open-24-hours">Open 24 hours</p>
+          </div>
+          :
+          <div>
+            <input type="time" defaultValue={this.props.getDayHours(day, "opens_at", index)} data-key={day} data-field="opens_at" onChange={(e) => this.props.handleScheduleChange(day, index, "opens_at", e.target.value)}/>
+            <input type="time" defaultValue={this.props.getDayHours(day, "closes_at", index)} data-key={day} data-field="closes_at" onChange={(e) => this.props.handleScheduleChange(day, index, "closes_at", e.target.value)}/>
+            {index > 0 ? <button onClick={() => this.props.removeTime(day, index)}className="remove-time">x</button> : ''}
+          </div>
+        }
       </li>
     );
   }
 
   render() {
+    let { dayHours } = this.props;
     let day = this.props.day;
+    let is24Hours = false;
+    let is24Toggle = null;
+
+    if (dayHours[0].opens_at === 0 && dayHours[0].closes_at === 2359) {
```

Comment:
This pattern seems to reoccur a lot. Not blocking this PR, but it'd be a good idea to create a class that represents a `ScheduleDay` and give it methods like `is24Hours()` and `toggle24Hours()` so that this logic isn't repeated everywhere.